crates/holochain_sqlite/src/store/p2p_agent_store.rs

```diff
@@ -645,13 +645,14 @@ impl P2pRecord {
 
         let signed_at_ms = signed.signed_at_ms;
         let expires_at_ms = signed.expires_at_ms;
-        let arc = signed.storage_arc;
+        let arq = signed.storage_arq;
 
-        let storage_center_loc = arc.start_loc().into();
+        let storage_center_loc = arq.start_loc().into();
 
         let is_active = signed.is_active();
 
-        let (storage_start_loc, storage_end_loc) = arc.to_primitive_bounds_detached();
+        let (storage_start_loc, storage_end_loc) =
+            arq.to_dht_arc_std().to_primitive_bounds_detached();
```

Comment:
maybe i missed it, or the changes naturally follow from the type changes, but I didn't see parallel updates to the p2p_agent_store cache in kitsune. Did that get handled?

Reply:
@neonphog the continuous arc representation still serves a purpose, especially for checking set membership. The p2p agent store is all about querying to see what is in what arcs, so it's good for that to be in continuous arc form.

The problematic part was converting from continuous to quantized form, but as long as we originate from and do our manipulations in the quantized form, then it's ok for data to wind up in continuous form as long as it doesn't get converted back.